episodes/manage_images.md

```diff
@@ -30,24 +30,64 @@ Objectives
 
 ## Exercises
 
-::::{dropdown} Specifying a custom image tag.
+::::{dropdown} Software Challenges
+:open:
+
+Think about the challenges that you may have faced with using software for your work.
+
+- Is there a tool that you wanted to use that was not compatible with your operating system?
+- Have you ever struggled to install a piece of software?
+- Did you and your collaborators fail to reproduce results using the same software application?
+
+Write up your thoughts in our shared collaborative document.
+
+::::
+
+::::{dropdown} Reasons to use containers
+:open:
+
+Which of the following statements are *True* or *False*?
+
+1. Containers are lightweight, portable and isolated units of software that can be used on any computer and operating system. 
+
+1. An image is a runnable, self-contained software environment and created from a container. A container is a shareable "recipe" used to create images.
+
+1. A container enables reproducible modelling or analyses to be carried out on your laptop or in the cloud.
+
+1. Containers are easily scalable and can be deployed on many machines to distribute work.
+
+:::{dropdown} Solution
+
+1. *True*
+
+1. *False* – A *container* is a runnable, self-contained software environment or service and created from an *image*. An *image* is a shareable "recipe" used to create *containers*.
+
+1. *True*
+
+1. *True*
+
+:::
+
+::::
+
+::::{dropdown} Specifying a custom image tag
 :open:
 
 Which of the following would you paste into the _Custom Image_ field to add the latest version of the `handbook-authoring-image` image to your hub?
 
 1. quay.io/2i2c/handbook-authoring-image:bbe4225a7940
 1. quay.io/2i2c/handbook-authoring-image
-1. docker pull quay.io/2i2c/handbook-authoring-image:ad18f6ea575d
-1. quay.io/2i2c/handbook-authoring-image:ad18f6ea575d
+1. `docker pull` quay.io/2i2c/handbook-authoring-image:b5b2c88daa24
+1. quay.io/2i2c/handbook-authoring-image:b5b2c88daa24
 
 :::{dropdown} Solution
-4. quay.io/2i2c/handbook-authoring-image:ad18f6ea575d
+4. quay.io/2i2c/handbook-authoring-image:b5b2c88daa24
 
 Notes on the other answers:
 
-1. This is using an older version tag and not the latest version.
+1. This is using an older version tag and not the latest version available.
```

Comment:
I really like this MCQ but are we creating a maintenance headache for ourselves? How often do you expect that the tag will change on this image?

Reply:
This is a really good point that I has also crossed my mind. Will have a think!

Reply:
I've written a GH action in https://github.com/czi-catalystproject/hub-champion-training/pull/27/commits/79da49df910bfb1a29665ba5a58c4746b2d815e8 that should update the exercise with the latest image tag by using the quay.io API

I'll merge this PR for now to trigger the GH action and if it fails will fix it up in a separate PR 👍

Reply:
Lol ofc nothing works first time round but the fix is here: https://github.com/czi-catalystproject/hub-champion-training/pull/36